# Goal Information ROS Message

string type
string base_obj
string target_obj
geometry_msgs/Pose pose